js/gui/gui-layout.js

```diff
@@ -395,6 +460,39 @@ function createLayout() {
         }
     });
 
+    //music
+        let songCounter = 0;
+        let musicboxbtn = document.getElementById("musicboxbutton")
+        musicboxbtn.innerHTML = getL("Music"); 
+        let musicbox = document.getElementById("musicbox") 
```

Comment:
For these music related elements, we might need to have a condition check to see if they exist before using them.